hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerStateMachineFailures.java

```diff
@@ -179,22 +194,6 @@ public void testContainerStateMachineFailures() throws Exception {
         .getDatanodeStateMachine().getContainer();
     Assert
         .assertNull(ozoneContainer.getContainerSet().getContainer(containerID));
-
-    OzoneKeyDetails keyDetails = objectStore.getVolume(volumeName)
-        .getBucket(bucketName).getKey("ratis");
-
-    /**
-     * Ensure length of data stored in key is equal to number of bytes written.
-     */
-    Assert.assertTrue("Number of bytes stored in the key is not equal " +
-        "to number of bytes written.", keyDetails.getDataSize() == written);
-
-    /**
-     * Pending data from the second write should get written to a new container
-     * during key.close() because the first container is UNHEALTHY by that time
-     */
-    Assert.assertTrue("Expect Key to be stored in 2 separate containers",
-        keyDetails.getOzoneKeyLocations().size() == 2);
   }
```

Comment:
I am creating a tool to aid developers by partially automating unti testing.
Could you speak to the validity of these generated assert statement for the given test method?

`org.junit.Assert.assertEquals( HddsProtos.getHddsDatanodes( result ), groupOutputStream);`
`org.junit.Assert.assertEquals( OmKeyLocationInfo.getHddsDatanodes(), groupOutputStream.getHddsDatanodes( rs ));`
`org.junit.Assert.assertEquals( OmKeyLocationInfo.getHddsDatanodes(), groupOutputStream.getHddsDatanodes( groupOutputStream ));`

Thank you.